packages/clay-empty-state/src/EmptyState.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+const emptyStateImg = require('../images/empty_state.gif');
+const searchStateImg = require('../images/search_state.gif');
+const successStateImg = require('../images/success_state.gif');
```

Comment:
So I am thinking that instead of actually supplying these images, we would just add a prop on the component for the path to the image.

```jsx
<ClayEmptyState imageSrc="/some/path/to.gif" si />
```

Reply:
@jbalsas what do you think about this, should the images be internal to the component itself?